### PR TITLE
Session state UX/UI improvements

### PR DESCRIPTION
**Pull request by jtamiace**
## Summary

Various UX improvements to the session/login/logout UI, including those on the following Trello cards:
https://trello.com/c/MOP04ZIq/378-bug-user-pop-out-widget-needs-modal-style-handling
https://trello.com/c/swe5Qy8H/380-enhance-log-in-click-area-should-be-entire-nav-block

- [x] Redirects to Learn on logout
- [x] Redirects to Manage for Admins/Superusers on login
- [x] Can click out of logout modal to close it
- [x] Session tab, login modal, user dropdown are all keyboard accessible

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

*If you PR has a significant size, give the reviewer some helpful remarks*

## Issues addressed

List the issues solved or partly solved by the PR

## Documentation

If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)

## Screenshots (if appropriate)

They're nice. :)

### CODE REVIEW

**Comment by jtamiace** [comment]
k im no longer touching this for now

File: kolibri/core/assets/src/vue/modal/index.vue [hunk 1]
```diff
@@ -44,12 +45,11 @@
   module.exports = {
 
     methods: {
-      toggleModal() {
-        if (!this.modalstate) {
-          this.togglemodal(true);
-        } else {
-          this.togglemodal(false);
-        }
+      closeModalHack() {
```
Comment (rtibbles) [inline kolibri/core/assets/src/vue/modal/index.vue:48]: Method names as TODOs?

File: kolibri/core/assets/src/vue/session-nav-widget/index.vue [hunk 2]
```diff
@@ -46,25 +46,50 @@
...
         }
-        return '?';
+        return this.kind;
       },
     },
     methods: {
-      showUserDropdown() {
-        if (this.showDropdown) {
-          this.showDropdown = false;
+      // extreme hack for making entire session tab clickable/accessible
```
Comment (rtibbles) [inline kolibri/core/assets/src/vue/session-nav-widget/index.vue:72]: X-Games contender right here.

Comment (aronasorman) [inline kolibri/core/assets/src/vue/session-nav-widget/index.vue:72]: Someone's following eXtreme programming: https://en.wikipedia.org/wiki/Extreme_programming

File: kolibri/plugins/management/assets/src/vue/index.vue [hunk 3]
```diff
@@ -47,12 +48,17 @@
...
+        if (this.kind[0] === UserKinds.SUPERUSER || this.kind[0] === UserKinds.ADMIN) {
+          return true;
+        }
+        return false;
+      },
     },
     vuex: {
       getters: {
         pageName: state => state.pageName,
-        isAdminOrSuperuser: state => state.core.is_admin_or_superuser,
```
Comment (rtibbles) [inline kolibri/plugins/management/assets/src/vue/index.vue:55]: Was this the only place that `is_admin_or_superuser` was used?

Comment (jtamiace) [inline kolibri/plugins/management/assets/src/vue/index.vue:55]: ..... it shouldn't be used at all. i'm going to purge this from the code one sec

Comment (jtamiace) [inline kolibri/plugins/management/assets/src/vue/index.vue:55]: It's also in setup_wizard at the moment

Comment (jtamiace) [inline kolibri/plugins/management/assets/src/vue/index.vue:55]: kk the purge is complete

File: kolibri/plugins/learn/assets/src/vue/content-unavailable-page/index.vue [hunk 4]
```diff
@@ -11,10 +11,21 @@
 
 <script>
 
+  const UserKinds = require('core-constants').UserKinds;
+
   module.exports = {
+
+    computed: {
+      isAdminOrSuperuser() {
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/vue/content-unavailable-page/index.vue:19]: This is moist, please make a Trello technical debt card about DRYing this up.

**Pull request merged**